app-sso/service-operators/tls-scenario-guides.hbs.md

```diff
@@ -0,0 +1,785 @@
+# Authserver TLS Scenario Guides
+
+`AuthServer` is a piece of security infrastructure. It is imperative to configure TLS for it, so that its _issuer
+URI_'s scheme is `https://`.
```

Comment:
**[for info, not blocking]**

OpenID Connect mandates TLS [spec](https://openid.net/specs/openid-connect-core-1_0.html#TLSRequirements)
OAuth 2 mandates TLS too [rfc](https://datatracker.ietf.org/doc/html/draft-ietf-oauth-v2-1-07#section-3.1)